lib/ansible/modules/source_control/github_deploy_key.py

```diff
@@ -21,6 +21,11 @@
   - "Adds or removes deploy keys for GitHub repositories. Supports authentication using username and password,
   username and password and 2-factor authentication code (OTP), OAuth2 token, or personal access token."
 options:
+  github_url:
+    description:
+      - The base URL of the GitHub API
+    required: false
```

Comment:
```suggestion
    required: false
    type: str
    version_added: "2.9"
```

Reply:
This module was added in 2.4. This PR is just an update.

Reply:
Fixed